app/governance/components/VotingInfoBox/VotingInfoBox.tsx

```diff
@@ -0,0 +1,152 @@
+
+  const [isHovered, setIsHovered] = useState(false);
+  const [isChecked, setIsChecked] = useState(false);
+  function getVotesInfo(
+    value: VoteOption,
+    votesData: VoteData
+  ): { votePercentage: string; voteAmount: string } {
+    if (value == VoteOption.YES) {
+      return { votePercentage: votesData.Yes, voteAmount: votesData.YesAmount };
+    }
+    if (value == VoteOption.NO) {
+      return { votePercentage: votesData.No, voteAmount: votesData.NoAmount };
+    }
+    if (value == VoteOption.ABSTAIN) {
+      return {
+        votePercentage: votesData.Abstain,
+        voteAmount: votesData.AbstainAmount,
+      };
+    }
+    if (value == VoteOption.VETO) {
+      return {
+        votePercentage: votesData.Veto,
+        voteAmount: votesData.VetoAmount,
+      };
+    }
+    return { votePercentage: "", voteAmount: "" };
+  }
+  //const color3 = `rgba(${color2},0.5)`;
+
+  const getHoverStyle = () => {
+    if (isSelected && isActive) {
+      return { backgroundColor: color1, cursor: "pointer" };
+    }
+    if (isHovered && isActive) {
+      return { backgroundColor: color2, cursor: "pointer" };
+    }
+    if (isHighest && !isActive) {
+      return { backgroundColor: color2 };
+    }
+    return {};
+  };
+  // const getCheckedStyle = ()=>{
+
+  // }
+
+  return (
+    <div
+      className={styles.proposalInfoVoting}
+      onMouseEnter={() => setIsHovered(true)}
+      onMouseLeave={() => {
+        setIsHovered(false);
+      }}
+      onClick={() => {
+        onClick();
+      }}
+      style={getHoverStyle()}
+    >
+      {isActive && (
+        <div className={styles.radioBtnContainer}>
+          <RadioButton
+            value={value}
+            isActive={isActive}
+            onClick={() => {
+              selectedVote = value;
+            }}
+            selectedValue={selectedVote}
+            checkedColor={color1}
+          ></RadioButton>{" "}
+        </div>
+      )}
+
+      <div className={styles.votingInfoRow1}>
+        <div
+          style={{
+            display: "flex",
+            flexDirection: "row",
+            justifyContent: "space-around",
+          }}
+        >
+          {" "}
+          <div
+            style={{ backgroundColor: color1, margin: "10px 5px 0px 10px" }}
+          ></div>{" "}
+          <div>
+            <Text font="proto_mono">{value}</Text>
+          </div>
+        </div>
+      </div>
+      <div className={styles.votingInfoRow2}>
+        <div className={styles.infoRow1First}>
+          <Text font="proto_mono">
+            {getVotesInfo(value, votesData).votePercentage}%
+          </Text>
+        </div>
+        <div className={styles.infoRow1First}>
+          <Container
+            direction="row"
+            gap={6}
+            center={{
+              vertical: true,
+            }}
+          >
+            <Text font="proto_mono" opacity={0.4} size="x-sm">
+              {formatBalance(getVotesInfo(value, votesData).voteAmount, 0, {
```

Comment:
use displayAmount instead. We will have really big numbers here

Reply:
Done. Replaced the function